### PR TITLE
Update org.gnome.Platform to 3.38

### PR DESCRIPTION
**Pull request by Inglebard**
- update shared-modules
- add intltool for polkit
- update pangox-compat source
- update polkit version
- update glu path
- add gtk2 for gtkglext
- add multiple patchs

Please note, this is the first time I "build" something for a public project. If something is wrong or missing, let me know.

### CODE REVIEW

**Comment by Inglebard** [comment]
The build seems to failed due to 404 on : https://anydesk.com/_static/img/screenshots/anydesk-v5-linux-main-screen-en-2e1b4e.png 
However this url is good for me.

**Comment by LeandroStanger** [comment]
bot, build

File: com.anydesk.Anydesk.json [hunk 1]
```diff
@@ -30,34 +31,49 @@
...
                 {
-                    "type": "archive",
-                    "url": "http://ftp.gnome.org/pub/GNOME/sources/pangox-compat/0.0/pangox-compat-0.0.2.tar.xz",
-                    "sha256": "552092b3b6c23f47f4beee05495d0f9a153781f62a1c4b7ec53857a37dfce046"
+                    "type" : "patch",
+                    "path" : "0001-Re-add-pango_x_get_shaper_map-it-is-still-used-in-th.patch"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "0002-disable-shaper.patch"
```
Comment (wjt) [inline com.anydesk.Anydesk.json:45]:
```suggestion
                    "paths" : [
                        "0001-Re-add-pango_x_get_shaper_map-it-is-still-used-in-th.patch",
                        "0002-disable-shaper.patch"
                    ]
```

File: com.anydesk.Anydesk.json [hunk 2]
```diff
@@ -92,6 +109,10 @@
                 {
                     "type": "patch",
                     "path": "gtkglext-gcc8.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "x11-libs_gtkglext_files_gtkglext-1.2.0-no-pangox.patch"
```
Comment (wjt) [inline com.anydesk.Anydesk.json:115]: As above, `"paths": []`

Comment (wjt) [reply com.anydesk.Anydesk.json:115]: If there is a patch here to remove a dependency on pangox, is pangox-compat still needed above?

**Review by wjt — CHANGES_REQUESTED** [review]
I agree with the bot, https://anydesk.com/_static/img/screenshots/anydesk-v5-linux-main-screen-en-2e1b4e.png is 404 for me. `com.anydesk.Anydesk.appdata.xml` will need to be updated with 1 or more valid screenshot URLs. I see 2/3 options:

1. Take a representative screenshot, following the guidelines in https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#qsr-app-screenshots-info, and add it to this repo.
2. Find a screenshot on the Anydesk website, and hotlink it
3. https://anydesk.com/en/press has a press kit which contains a bunch of screenshots in a ZIP file. If any of them are appropriate (they seem to all be on Windows but maybe the Linux app looks the same), add them to this repo and use those

A tip: if you add the screenshot, commit it, and push that commit to your fork, then you can use a GitHub raw URL to the file embedding the commit ID and it'll work even though the branch is not merged.

**Comment by wjt** [comment]
Thanks for this hard work!

**Comment by Inglebard** [comment]
Hi @wjt,
pangox seems still required at runtime:
_/app/extra/anydesk: error while loading shared libraries: libpangox-1.0.so.0: cannot open shared object file: No such file or directory_

The actual screenshot is present in the official website : https://anydesk.com/fr/downloads/linux. I don't understand why there is a 404.

The screenshots present inside https://anydesk.com/en/press don't represent the actual look of the application under linux. The only screenshot available is https://anydesk.com/_static/img/screenshots/anydesk-v5-linux-main-screen-en-2e1b4e.png. If it doesn't work, I will made homemade screenshot.

**Comment by wjt** [comment]
The screenshot on that page is https://anydesk.com/_static/img/screenshots/anydesk-v5-linux-main-screen-en-a5abed.png – a different URL.



I don't think it is a good screenshot because it includes stuff around the main window. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#qsr-app-screenshots-info says:

> Screenshots should not show anything outside the application's windows (including the background wallpaper). If you are taking a screenshot of the whole window, use your screenshot tool's "window" mode (including any window borders in the screenshot, and ensuring that the resulting image has an alpha mask for the rounded corners).

**Comment by Inglebard** [comment]
Ok, I got a 404 with your URL.

For an unknow reason, Anydesk seems to show (and filter) different screenshot base on location.

The "anydesk-v5-linux-main-screen-en-2e1b4e.png" image is the same without the fake laptop.
I will upload it on the repository.

File: com.anydesk.Anydesk.appdata.xml [hunk 3]
```diff
@@ -15,7 +15,7 @@
   <description xml:lang="es"><p>Gracias a AnyDesk, contará con conexiones seguras y fiables de escritorio remoto para profesionales informáticos y usuarios en movimiento.</p><p>NOTA: Esta aplicación no ha sido empaquetado por AnyDesk</p></description>
   <screenshots>
     <screenshot>
-      <image type="source">https://anydesk.com/_static/img/screenshots/anydesk-v5-linux-main-screen-en-2e1b4e.png</image>
+      <image type="source">https://github.com/Inglebard/com.anydesk.Anydesk/raw/master/Screenshots/anydesk-v5-linux-main-screen-en-2e1b4e.png</image>
```
Comment (wjt) [inline com.anydesk.Anydesk.appdata.xml:18]:
```suggestion
      <image type="source">https://github.com/flathub/com.anydesk.Anydesk/raw/b27de5e35d6651a35a2ea8e2568dfc4d021c4b0c/Screenshots/anydesk-v5-linux-main-screen-en-2e1b4e.png</image>
```

**Review by wjt — APPROVED** [review]
Looks good to me. I successfully connected my desktop running this branch to my Android phone, in both directions.

**Pull request merged**